client/my-sites/marketing/ultimate-traffic-guide/index.js

```diff
@@ -0,0 +1,295 @@
+				) }
+			</p>
+			<p>
+				{ translate(
+					'If you’ve spent any time looking for marketing help, you know that marketing guides like this often cost $100 or more. But you won’t have to pay anywhere near that amount.'
+				) }
+			</p>
+
+			{ isLoading ? (
+				<Placeholder />
+			) : (
+				<p>
+					<strong>
+						{ translate(
+							'In fact, if you act today you’ll pay just %(cost)s for The Ultimate Traffic Guide.',
+							{
+								args: { cost },
+							}
+						) }
+					</strong>
+				</p>
+			) }
+		</>
+	);
+
+	const treatmentVariation = () => (
+		<>
+			<h1 className="ultimate-traffic-guide__header">
+				{ translate(
+					'Discover the most important traffic secrets used by our most successful customers'
+				) }
+			</h1>
+			<h2 className="ultimate-traffic-guide__sub-header">
+				{ translate( 'Claim your copy of the Ultimate Traffic Guide today' ) }
+			</h2>
+			<p>
+				{ translate(
+					'Do you want more traffic to your website?' +
+						' Of course you do, who builds a website and doesn’t want traffic?' +
+						' You’ve time building your website and you’ve just realized that “if you build it, they will come” does not always apply.'
+				) }
+			</p>
+			<p>
+				{ translate(
+					'Our marketing team developed The Ultimate Traffic Guide to help WordPress.com users learn some of the same tactics and strategies our most successful customers use to build their audiences. ' +
+						'This easy-to-follow and beginner-friendly guide reveals today’s best strategies and tactics for attracting more visitors.'
+				) }
+			</p>
+			<p>
+				{ translate( 'Here are just some of the topics we cover in The Ultimate Traffic Guide:' ) }
+			</p>
+			<ul>
+				<li>
+					{ translate(
+						'{{b}}Use Your Traffic to Build Even More Traffic{{/b}} The Ultimate Traffic Guide covers how to use your own site stats to further increase your traffic, keep visitors on your site longer, and convert traffic into qualified leads.',
+						{
+							components: { b: <b /> },
+						}
+					) }
+				</li>
+				<li>
+					{ translate(
+						'{{b}}Leverage Social Media to Expand Your Reach{{/b}} Know where your ideal audience spends time and learn how to leverage those social channels to increase traffic.',
+						{
+							components: { b: <b /> },
+						}
+					) }
+				</li>
+				<li>
+					{ translate(
+						'{{b}}Keeping the Engagement Going{{/b}} we’ll share how to choose the right newsletter service, write compelling content, build your list, and monitor key performance indicators to manage the critical ongoing relationship with your site visitors.',
+						{
+							components: { b: <b /> },
+						}
+					) }
+				</li>
+			</ul>
+			<p>
+				{ translate(
+					'You’re going to want to print out all 96 pages of the Ultimate Traffic Guide and keep it by your desk, because you’re going to consult this guide often.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Print it out and keep it by your desk, because you’re going to consult this guide often.' )` *ES Score: 6*